### PR TITLE
feat: Layout API SDK addition

### PR DESCRIPTION
**Pull request by srikanth597**

### CODE REVIEW

File: src/changes.rs [hunk 1]
```diff
@@ -46,7 +46,7 @@ pub struct Changes {
 }
 
 // A utility function to generate the query string for multiple parameters
-fn generate_query_string(params: &HashMap<&str, Option<impl ToString>>) -> String {
```
Comment (kymppi) [inline src/changes.rs:49]: Could this function be moved to another file? Maybe something like `utils`.

**Review by kymppi — CHANGES_REQUESTED** [review]

File: src/changes.rs (hunk 1 above)
Comment (srikanth597) [reply src/changes.rs:67]: moved it to utils

**Comment by srikanth597** [comment]
@kymppi now we should be good

**Review by kymppi — CHANGES_REQUESTED** [review]
Oops sorry for having to deny this. The CI (tests) seem to report some problems so please fix those 🙂

**Comment by srikanth597** [comment]
now we are good with the checks, please review & merge it.

**Comment by srikanth597** [comment]
@kymppi also requesting you to add hactoberfest labels

**Review by kymppi — APPROVED** [review]

**Pull request merged**